src/AbstractFixer.php

```diff
@@ -45,6 +46,13 @@ public function __construct()
         }
     }
 
+    public function fix(\SplFileInfo $file, Tokens $tokens)
```

Comment:
I am a side-project maintainer: I ended up having business logic on `fix`, but had to implement `applyFix`: what should have I done?
If `fix` would be final, I would got two parse error, and the rename would be easy.